modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -0,0 +1,104 @@
+* Have names prepared to assign to namespaces, on both the source cluster and the destination cluster, in which to run the VPN tunnel. These namespaces should not be created in advance. For information on namespace rules, see \https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names.
+* When connecting multiple firewall-protected source clusters to the cloud cluster, each source cluster requires its own namespace.
+* OpenVPN server is installed on the destination cluster.
+* OpenVPN client is installed on the source cluster.
+* When configuring the source cluster in {mtc-short}, the API URL takes the form of https://proxied-cluster.${namespace}.svc.cluster.local.
+  See https://docs.openshift.com/container-platform/4.9/migration_toolkit_for_containers/advanced-migration-options-mtc.html#migration-migrating-applications-api_advanced-migration-options-mtc, step 3: _Create a MigCluster CR manifest for each remote cluster_ or...there's a web console way to do this to, but we can't find the documentation for it.
+* The `crane` utility is installed. (??? Need to add a topic on how to install it. ???)
+* The MTC web console and Migration Controller must be installed on the target cluster.
+
+Replace ${namespace} with either openvpn or the namespace you specified when running the command to set up the tunnel.
+
+.Procedure
+
+// ??? "Command system" is a placeholder for the system issuing the `tunnel-api` command. ???
+
+. Log in remotely to a node on the source cluster and a node on the destination cluster
+
+. Obtain the cluster-context for both clusters after logging in. ???HOW???
+
+. Establish a tunnel by entering the following command on the command system:
++
+[source,terminal]
+----
+$ crane tunnel-api [--namespace _<namespace>_] \
+      --destination-context _<destinatin-cluster>_ \
+      --source-context _<source-cluster>_
+----
++
+If you don't specify a namespace, the command uses the default value `openvpn`.
+
+For example:
++
+[source,terminal]
+----
+$ crane tunnel-api --namespace my_tunnel \
+      --destination-context openshift-migration/c131-e-us-east-containers-cloud-ibm-com/admin \
+      --source-context default/192-168-122-171-nip-io:8443/admin
+----
++
+The command generates SSL Certificates. This process might take several minutes. A message appears when the process completes.
++
+The OpenVPN server starts on the destination cluster and the OpenVPN client starts on the source cluster.
++
+After a few minutes, the load balancer resolves on the source node.
+[TIP]
+====
+You can view the log (???WHICH LOG?) to check the status of this process by entering the following commands with root privileges (???WHAT ExACTLY ARE THESE COMMANDS? `oc get po` is not in the CLI reference):
+
+[source,terminal]
+----
+# oc get po -n _<namespace>_
+NAME            READY     STATUS      RESTARTS    AGE
+???What should go here???
```

Comment:
A list of pods in the namespace, with their status, restarts, and age,

Reply:
The video demo shows the result. Source cluster at 2:12.

Reply:
Thanks. I added that, but I'm not sure how to refer to the string following `openvpn`. I referred to is `<log-hash>`. @jmontleon Can you weigh in?

Reply:
Fixed.